docs/docs/genai/tracing/quickstart.mdx

```diff
@@ -0,0 +1,323 @@
+
+### Configure OpenAI API Key
+
+Set your OpenAI API key as an environment variable:
+
+```python
+import os
+
+# Set your OpenAI API key
+os.environ["OPENAI_API_KEY"] = "your-api-key-here"  # Replace with your actual API key
+```
+
+:::tip
+You can also set the environment variable in your shell before running the script:
+
+```bash
+export OPENAI_API_KEY="your-api-key-here"
+export MLFLOW_TRACKING_URI="http://your-mlflow-server:5000"  # Optional: for remote server, set as 'databricks' if connecting to a Databricks account
+```
+:::
+
+## Step 2: Create a simple application with tracing
+
+```python
+import mlflow
+import openai
+from openai import OpenAI
+import os
+
+# Set up MLflow experiment
+mlflow.set_experiment("openai-tracing-quickstart")
+
+# Enable automatic tracing for all OpenAI API calls
+mlflow.openai.autolog()
+
+client = OpenAI()
+
+
+def get_weather_response(location):
+    """Get a weather-related response for a given location."""
+    response = client.chat.completions.create(
+        model="gpt-4o-mini",
+        messages=[
+            {"role": "system", "content": "You are a helpful weather assistant."},
+            {"role": "user", "content": f"What's the weather like in {location}?"},
+        ],
+        max_tokens=100,
+        temperature=0.7,
+    )
+    return response.choices[0].message.content
+
+
+# Execute the traced function
+location = "San Francisco"
+response = get_weather_response(location)
+
+print(f"Query: What's the weather like in {location}?")
+print(f"Response: {response}")
+print("\nTraces have been captured!")
+print(
+    "View them in the MLflow UI at: http://127.0.0.1:5000 (or your MLflow server URL)"
+)
+```
+
+## Step 3: Run the application
+
+<Tabs>
+<TabItem value="notebook" label="In a Jupyter Notebook" default>
+
+Simply run the code cell above. You should see output similar to:
+
+```
+Query: What's the weather like in San Francisco?
+Response: I don't have real-time weather data, but San Francisco typically has mild temperatures year-round...
+Traces have been captured!
+View them in the MLflow UI at: http://127.0.0.1:5000
+```
+
+:::tip
+If you're using Jupyter with MLflow 2.20+, the trace UI will automatically display in your notebook when traces are generated!
+:::
+
+</TabItem>
+
+<TabItem value="script" label="As a Python Script">
+
+1. Save the code above to a file named `weather_app.py`
+2. Run the script:
+
+```bash
+python weather_app.py
+```
+
+You should see output similar to:
+
+```
+Query: What's the weather like in San Francisco?
+Response: I don't have real-time weather data, but San Francisco typically has mild temperatures year-round...
+Traces have been captured!
+View them in the MLflow UI at: http://127.0.0.1:5000
+```
+
+</TabItem>
+</Tabs>
+
+## Step 4: Explore traces in the MLflow UI
+
+1. **Open the MLflow UI**:
+   - If running locally: Navigate to http://127.0.0.1:5000
+   - If using remote server: Navigate to your MLflow server URL
```

Comment:
The bullet point doesn't render, so this part looks a bit weird
<img width="919" alt="image" src="https://github.com/user-attachments/assets/d653aaa8-0b71-4de8-9e63-def7cf0aa9f0" />